### PR TITLE
Adding emojis section to special characters.

### LINKED ISSUE #13348: Add emojis to special characters
Add a dedicated emoji set to the special characters feature.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Feature (special-characters): Added the "Emojis" category in the special characters feature. Closes #13348

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: packages/ckeditor5-special-characters/src/specialcharactersemojis.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module special-characters/specialcharactersarrows
```
Comment (pomek) [inline packages/ckeditor5-special-characters/src/specialcharactersemojis.js:7]:
```suggestion
 * @module special-characters/specialcharactersemojis
```

**Comment by godai78** [comment]
Follow-up: update the \[special characters\](https://ckeditor.com/docs/ckeditor5/latest/features/special-characters.html) guide with a new example other than emojis.

**Review by pomek — CHANGES_REQUESTED** [review]
As mentioned. Also, we should rewrite [the documentation](https://ckeditor.com/docs/ckeditor5/latest/features/special-characters.html#adding-a-new-category) in the PR to avoid having two "Emoji" plugins.

File: packages/ckeditor5-special-characters/src/specialcharactersemojis.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+		const t = editor.t;
+
+		editor.plugins.get( 'SpecialCharacters' ).addItems( 'Emojis', [
+			{ title: 'smiley face', character: '😊' },
```
[the remark this answers is not in the document]
Comment (godai78) [reply packages/ckeditor5-special-characters/src/specialcharactersemojis.js]: I have never encountered translated emoji titles. Quick additional research shows no confirmed case of an app using translated emoji titles, at least for Poland.

**Review by pomek — COMMENTED** [review]
The code looks good. Now we need to update the documentation.

File: packages/ckeditor5-special-characters/docs/features/special-characters.md [hunk 3]
```diff
@@ -31,26 +31,28 @@ By default, a few categories of special characters have been defined. You can ea
...
-		{ title: 'heart', character: '❤️' }
+	editor.plugins.get( 'SpecialCharacters' ).addItems( 'Borders', [
+		{ title: 'horizontal', character: ' ━ ' },
+		{ title: 'vertical', character: '┃' },
+		{ title: 'top left', character: '┏' },
+		{ title: 'top right', character: '┓' },
+		{ title: 'bottom left', character: '┗' },
+		{ title: 'bottom right', character: '┛' },
+		{ title: 'cross', character: '╋' }
 	], { label: 'Emoticons' } );
```
Comment (pomek) [inline packages/ckeditor5-special-characters/docs/features/special-characters.md:49]: This should be renamed to `Borders` instead of `Emoticons`

**Comment by apadol-cksource** [comment]
The change may be in conflict with plugins developed by the community in terms of UX, for instance, [https://www.npmjs.com/package/@phudak/ckeditor5-emoji](https://www.npmjs.com/package/@phudak/ckeditor5-emoji). Let's wait with merging it until we discuss the general approach to emojis. cc @Reinmar

**Pull request closed without merging**